src/module.ts

```diff
@@ -26,11 +27,15 @@ export default defineNuxtModule<ModuleOptions>({
     nuxt.options.build.transpile ||= []
     nuxt.options.build.transpile.push('vue-clerk', '@clerk/clerk-js')
 
+    const { installServerMiddleware = true } = options
+
     addTemplate({
       filename: 'nuxt-clerk.mjs',
+      write: true,
       getContents: () => [
         'export default {',
-        ` vueClerkConfig: ${options.vueClerk ? JSON.stringify(options.vueClerk) : 'null'}`,
+        ` vueClerkOptions: ${options.vueClerk ? JSON.stringify(options.vueClerk) : 'null'},`,
+        ` h3ClerkOptions: ${options.h3Clerk ? JSON.stringify(options.h3Clerk) : 'null'},`,
```

Comment:
I assume this will not be included in client bundle 👀

Reply:
It all depends on where it is imported! Let me check

Reply:
@RodrigoProjects would you be interested in doing that?